index.html

```diff
@@ -18,287 +19,312 @@
         </script>
     </head>
     <body>
-        <div class="container">
-            <header>
-                <nav>
-                    <ul class="nav-items-ul flex">
-                        <li id="my-logo-nav-item"><a href="#logo">Dyary Raoof
-                                BayzAgha</a></li>
-                        <li class="left-nav-items"><a href="#work-area-grid">Portfolio</a></li>
-                        <li class="left-nav-items"><a href="#about-section">About</a></li>
-                        <li class="left-nav-items">
-                            <a href="#contact-section-container">
-                                Contact
-                            </a>
-                        </li>
-                        <li id="hamburger-nav-item">
-                            <a href="#">
-                                <img src="images/hamburger-icon.png"
-                                    alt="hamburger
-                                    icon">
-                            </a>
-                        </li>
-                    </ul>
-                </nav>
-            </header>
+        <div class="body-wrapper">
+            <nav id="popup-nav" class="display-none">
```

Comment:
- [ ] Your mobile menu is not working as expected. When the mobile app is launched the screen shows the _overlay_ without the hamburger menu. Kindly ensure to improve upon this aspect.
- [ ]  Your mobile menu is missing the blurred background feature beneath the mobile menu overlay. Kindly ensure to improve upon this aspect, as we want our design to as much as possible closely resemble the provided design in this [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- [ ] The `cancel` button on top of the mobile menu is not working. Please ensure to improve upon this feature

**Your mobile screen design:**
![image](https://user-images.githubusercontent.com/12745474/131416812-97c6bb2e-f1f8-4caf-92bb-a79c8ef4af50.png)

**The original mobile screen design:**
![image](https://user-images.githubusercontent.com/12745474/131416978-bade47d0-1a99-4d69-8051-214df80c72bf.png)